### PR TITLE
fix(poke): use super user rights to view other workspace's DS

### PR DESCRIPTION
**Pull request by fontanierh**
forgot to give the right role

### CODE REVIEW

File: front/lib/auth.ts [hunk 1]
```diff
@@ -49,9 +49,15 @@ export class Authenticator {
    *
    * @param session any NextAuth session
    * @param wId string target workspace id
+   * @param asSuperUser boolean if true, will return an admin role for the user if the user is a
+   * superuser
    * @returns Promise<Authenticator>
    */
-  static async fromSession(session: any, wId: string): Promise<Authenticator> {
+  static async fromSession(
```
Comment (PopDaph) [inline front/lib/auth.ts:56]: NIT: could be an object over a list of params 🙇🏻‍♀️ 
(feel free to discard)

**Review by PopDaph — APPROVED** [review]

File: front/lib/auth.ts (hunk 1 above)
Comment (fontanierh) [reply front/lib/auth.ts:56]: 😬

**Comment by spolu** [comment]
This is quite scary. Todo for us as soon as we get back is to design safer. 

Adding an extra argument by mistake may easily lead to us leaking our user data on the internet. 

Have you checked that no fromSession already has an extra argument?

**Comment by spolu** [comment]
I now see that the user is checked internally. 

Maybe having an explicit function fromSuperUserSession would make things much clearer that we’re overriding as super user for this particular session

**Pull request merged**